src/test/java/uk/gov/hmcts/reform/sscs/ccd/deserialisation/SscsCaseCallbackDeserializerTest.java

```diff
@@ -152,4 +152,18 @@ public void should_deserialize_direction_issed_callback() throws IOException {
         assertEquals("DirectionNotice.pdf", sscsInterlocDirectionDocument.getDocumentFileName());
         assertEquals("Direction Notice", sscsInterlocDirectionDocument.getDocumentType());
     }
+
+    @Test
+    public void should_deserialize_info_request_callback() throws IOException {
+        sscsCaseCallbackDeserializer = new SscsCaseCallbackDeserializer(mapper);
+
+        String path = getClass().getClassLoader().getResource("infoRequest.json").getFile();
+        String json = FileUtils.readFileToString(new File(path), StandardCharsets.UTF_8.name());
+
+        Callback<SscsCaseData> actualSscsCaseCallback = sscsCaseCallbackDeserializer.deserialize(json);
+
+        String informationFromAppellant = actualSscsCaseCallback.getCaseDetails().getCaseData().getInformationFromAppellant();
+
+        assertEquals("No", informationFromAppellant);
+    }
```

Comment:
This test just checks jackson is working under the hood don't think we need to do that. I would have said the original tests Jack wrote covered this class fine.